### DISCUSSION TITLE
Location of files in directory (folder)

### DISCUSSION THREAD

**Discussion by bobthiel** [zachowj/hass-node-red#98] [Q&A] [answered]
Location of files in directory (folder) in this repository.  I could not find the files??

Thanks,
Bob

**Comment by zachowj** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/37859597/147836921-23ce3cc7-1400-4503-868c-f9008e277525.png)

**Comment by bobthiel** [comment]
Very fast response, busy now, but will try later today.

**Reply by bobthiel** [reply]
Found files, working fine.  Thanks
